cmd/capmcd/bmcapi.go

```diff
@@ -315,13 +316,33 @@ func (d *CapmcD) doBmcPowerCall(call bmcCall) bmcPowerRc {
 	}
 
 	var body string
+	var sessionAuthPath = ""
+	var sessionAuthBody = ""
 	switch ni.Type {
 	// The CabinetPDUOutlet HMSType has been depricated in favor of
 	// CabinetPDUPowerConnector. Support both for now.
 	case "CabinetPDUOutlet":
 		fallthrough
 	case "CabinetPDUPowerConnector":
-		body = fmt.Sprintf(`{"PowerState": "%s"}`, resetType)
+		var HPEPDU = true
+		if strings.Contains(ni.BmcFQDN, "rts") {
+			HPEPDU = false
+		}
+		if HPEPDU {
+			outletNum := strings.Split(ni.Hostname, "v")
+			if len(outletNum) < 2 {
+				log.Printf("Could not get outlet number")
+				outletNum = append(outletNum, "")
```

Comment:
Not sure what is going on here, why append an empty zero length string twice?

Reply:
Just returns now if outlet unavailable